src/System.IO.Ports/src/System/IO/Ports/InternalResources.Windows.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why does Ports need its own copy of this?  Why can't it use Common\CoreLib\System\IO\Win32Marshal.cs?

Reply:
I moved it from InternalResources.cs. I didn't know there were more copies